apollo-backend/src/main/java/io/logz/apollo/dao/BlockerDefinitionDao.java

```diff
@@ -13,4 +14,7 @@
     void addBlockerDefinition(BlockerDefinition blockerDefinition);
     void updateBlockerDefinition(BlockerDefinition blockerDefinition);
     void deleteBlockerDefinition(int id);
+    List<Integer> getOverrideBlockersIdsByUserEmail(String userEmail);
```

Comment:
Maybe lets add a "isUserAllowedOverrideOnBlocker(String userEmail, int blockerId) as well?
You are just fetching all from DB and iterate them, while this can be easily achieved in SQL